### PR TITLE
Allow to provide release channel in the release script

### PR DESCRIPTION
**Pull request by asaarilahti**
Replace hardcoded value with an argument.

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -1,25 +1,39 @@
 #!/usr/bin/env bash
-set -eux
+set -euo pipefail
 
-if [ -z "$1" ]
+if [ -z "${1:-}" ]
```
Comment (tkrop) [inline release.sh]: This `${1:-}` is semantically equal to `${1}`. Why do you propose to add `:-`?

**Review by tkrop — DISMISSED** [review]

File: release.sh (hunk 1 above)
Comment (asaarilahti) [reply release.sh]: We are using `set -u`, so `${1}` will result in `./release.sh: line 4: 1: unbound variable` when the argument is not provided. This is an existing issue, just fixed now.

Comment (cberg-zalando) [reply release.sh]: I typically use a construct like

```
if [ $# -lt 1 ]
```

File: release.sh [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
+if [ -z "${JETBRAINS_HUB_TOKEN:-}" ]
   then
     echo "Please put JETBRAINS_HUB_TOKEN in .env: JETBRAINS_HUB_TOKEN=<token>"
     exit 1
 fi
 
 NEW_VERSION=$1
 
-sed -i "s/<version>.*<\/version>/<version>$NEW_VERSION<\/version>/g" src/main/resources/META-INF/plugin.xml
+sed -i '' "s/<version>.*<\/version>/<version>$NEW_VERSION<\/version>/g" src/main/resources/META-INF/plugin.xml
```
Comment (cberg-zalando) [inline release.sh]: What do these two `''`actually do?

File: release.sh (hunk 1 above)
Comment (asaarilahti) [reply release.sh]: Hmm, that will allow empty strings `""` as an argument. How about something like https://github.com/zalando/intellij-swagger/pull/353/commits/45ff208327f68d9c204a6977eaa704c4772b23aa

Comment (cberg-zalando) [reply release.sh]: Ah yes. That's fine. I also tend to do these checks then also later to verify that no empty arguments are provided. So all is fine.

File: release.sh (hunk 2 above)
Comment (asaarilahti) [reply release.sh]: With Mac the extension is compulsory: `-i extension`, and empty value disables the backup. However, empty value does not seem to work in sed bundled in Ubuntu, so I'll try to see what is best option here.

**Comment by cberg-zalando** [comment]
As far as I remember, the BSD version of `sed` used by OS X does not support the `-i` parameter. So what I remember is, that you typically call `sed` with `-e` on the file and redirect the input to a temporary file and the use `mv` to overwrite the old one. This works in both worlds.

**Review by cberg-zalando — APPROVED** [review]

**Comment by asaarilahti** [comment]
> As far as I remember, the BSD version of `sed` used by OS X does not support the `-i` parameter.

`-i` is supported in Mac and Ubuntu, so that's probably enough. How about this? https://github.com/zalando/intellij-swagger/pull/353/commits/bf6f004e7160f547cad5999c5d4cbd30e285afbc

**Pull request merged**